[2022-05-06 11:15:19,249] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 manual__2022-05-06T05:44:33.761637+00:00 [queued]>
[2022-05-06 11:15:19,253] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 manual__2022-05-06T05:44:33.761637+00:00 [queued]>
[2022-05-06 11:15:19,253] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:15:19,253] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-05-06 11:15:19,253] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:15:19,260] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_3> on 2022-05-06 05:44:33.761637+00:00
[2022-05-06 11:15:19,262] {standard_task_runner.py:52} INFO - Started process 1464789 to run task
[2022-05-06 11:15:19,263] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', 'manual__2022-05-06T05:44:33.761637+00:00', '--job-id', '23', '--raw', '--subdir', '/home/sanjeev/Documents/SubDAGs/dags/paralle_processing.py', '--cfg-path', '/tmp/tmpzc42nw93', '--error-file', '/tmp/tmpppjygi89']
[2022-05-06 11:15:19,263] {standard_task_runner.py:77} INFO - Job 23: Subtask task_3
[2022-05-06 11:15:19,288] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_3 manual__2022-05-06T05:44:33.761637+00:00 [running]> on host kumar
[2022-05-06 11:15:19,320] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-05-06T05:44:33.761637+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-06T05:44:33.761637+00:00
[2022-05-06 11:15:19,321] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-06 11:15:19,321] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 5']
[2022-05-06 11:15:19,324] {subprocess.py:85} INFO - Output:
[2022-05-06 11:15:24,325] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-06 11:15:24,347] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20220506T054433, start_date=20220506T054519, end_date=20220506T054524
[2022-05-06 11:15:24,368] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-06 11:15:24,382] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
